Jenkins/Pipelines/TerraformPlanOnly/Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+#!groovy
+
+pipeline {
+  agent any
+
+  stages {
+    stage('FetchCode'){
+      steps {
+        checkout scm
+        dir('Secrets') {
+            sh 'aws s3 cp s3://cdis-terraform-state/planx-pla.net/v1/config.tfvars config.tfvars'
```

Comment:
what does version mean here?

Reply:
You're working on VPC migration.
I thought a good strategy to migrate is to do something like this:
* setup a v2/ VPC
* copy state from v1/ VPC
* tear down v1/ VPC

So the v2/ VPC starts with an empty state.

Reply:
if we want to use a version number, we can use a git tag?

Reply:
yeah - that's cool.  That's just what I was thinking when I set things up to save state there.  What would the git tag correspond to?  We may not want to migrate to a new VPC for every change to our terraform config - some things (like add elasitc search) we can just do in place ...

Reply:
would be corresponding to each release (which we haven't done any yet). Yeah we don't need to migrate new vpc for every change, I guess when a pr is merged, the changes can be applied to the test vpc `v1.1` and when we create a new git tag jenkins can do a actual vpc swap